exchange/exchange-ps/exchange/New-RetentionPolicy.md

```diff
@@ -117,10 +117,12 @@ The IsDefault switch specifies that this retention policy is the default retenti
 
 When you use this switch, you'll get a warning that this retention policy will replace the current default retention policy (the policy where the IsDefault property value is currently True) as the default. By default, the default retention policy is named Default MRM Policy.
 
-**Note**: Changing which retention policy is the default might or might not affect new and existing mailboxes based on the value of the RetentionPolicy parameter on the Set-MailboxPlan cmdlet:
+**Note**: Changing which retention policy is the default might or might not affect new and existing mailboxes based on the value of the RetentionPolicy parameter on the all mailbox plans:
```

Comment:
```suggestion
**Note**: Changing which retention policy is the default might or might not affect new and existing mailboxes based on the value of the RetentionPolicy parameter on all mailbox plans:
```

Reply:
@JohanFreelancer9 @chrisda sorry, I didn't realize I could commit the suggestion directly and made this change myself.  I'm not sure how to fix this conflict.